packages/angular_devkit/build_angular/test/protractor/works_spec_large.ts

```diff
@@ -32,9 +32,9 @@ describe('Protractor Builder', () => {
 
   it('overrides protractor specs', (done) => {
     host.scopedSync().rename(normalize('./e2e/app.e2e-spec.ts'),
-      normalize('./e2e/renamed-app.e2e-spec.ts'));
+      normalize('./e2e/renamed-app.e2e.specs.ts'));
```

Comment:
This test was always passing because `./e2e/renamed-app.e2e-specs.ts` always matched `src/**/*.e2e-spec.ts` which is configured in `protractor.conf.js`